### DISCUSSION TITLE
Hide Copilot Status

### DISCUSSION THREAD

**Discussion by KDonnelly33** [community/community#74980] [Copilot Conversations]
### Select Topic Area

General

### Body

I clicked on the bottom right to hide the the copilot staus icon. I can not figure out how to get it back.

**Comment by Akash1134** [comment]
Hi @KDonnelly33 , 

Here's what you can try to get the status icon back:

- Press  `Ctrl+Shift+P` on Windows or `Cmd+Shift+P` on Mac. (VSCode)
- Type "GitHub Copilot: Show Status Bar Icon" in the search bar and select the option from the list.

Hope that helps 🙂

**Comment by sijanpaudel14** [comment]
Response #1
